torch/csrc/jit/passes/freeze_module.cpp

```diff
@@ -220,11 +220,11 @@ class AttributePropagator {
           blocks.push(sub_block);
         }
 
-        // Modules with prim::ModuleDictIndex cannot be frozen because they
+        // Modules with prim::ModuleContainerIndex cannot be frozen because they
```

Comment:
We should try to get this to work. @gmagogsfm sometimes asks about what frontend team can do with perf. I think there's a lot to be said having frontend people work so that the new abstractions they introduce dont slow down user models. The alternative is that perf people stop making forward progress and begin working so that we dont regress.

Reply:
> We should try to get this to work.

What do you mean by this? We should try to get it to work without a custom operator?

Reply:
No, just add reasoning about the operator within freezing. Let's wait till we actually get an issue about this to do it.